README.md

```diff
@@ -76,16 +76,23 @@ Settings for included attachments.
-| Password           | string | Use this password to log in to the SMTP server.                                                                                                                             | password                 |
-
+| Property           | Type     | Description                                                                                                                                                                 | Example                  |
+|--------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Smtp server        | string   | SMTP server address.                                                                                                                                                        | smtp.somedomain.com      |
+| Port               | int      | SMTP server port.                                                                                                                                                           | 25                       |
+| SecureSocket       | enum     | Choose what SecureSocketOptions to use, default: SecureSocketOptions.Auto                                                                                                   | SecureSocketOptions.None |
+| AcceptAllCerts     | bool     | Accept all certificates when connecting the host, if true, will accept event invalid certificates. If false, will accept self-signed certificates if the root is untrusted. | false                    |
+| SkipAuthentication | bool     | If true, task will not authenticate to the SMTP server.                                                                                                                     | false                    |
+| Username           | string   | Use this username to log in to the SMTP server.                                                                                                                             | user                     |
+| Password           | string   | Use this password to log in to the SMTP server.                                                                                                                             | password                 |
+| CustomHeaders      | Header[] | Array of custom headers which will be added to sending of email. Can be left empty if no custom headers are needed. See Header.                                             | Check Header             |
+
+### Header
+
+| Property | Type   | Description                 | Example                   |
+|----------|--------|-----------------------------|---------------------------|
+| Key      | String | Key of the custom header.   | Content-Transfer-Encoding |
```

Comment:
In previous docs you have `string`, but in this place you have `String`. I think it is better to use same style everywhere.